e2e/cypress/tests/integration/enterprise/oauth/oauth_spec.ts

```diff
@@ -183,7 +188,9 @@ describe('Integrations page', () => {
                     data: {
                         appID: clientID,
                         appSecret: clientSecret,
-                    }});
+                    },
+                    waitFor: null,
```

Comment:
Should I probably change `waitFor` as an optional parameter in the `postIncomingWebhook` function instead of setting null values in multiple places?

Reply:
> Should I probably change `waitFor` as an optional parameter in the `postIncomingWebhook` function instead of setting null values in multiple places?

That would be great

Reply:
Done